src/extensions/fixtures/fixtures_loader.js

```diff
@@ -11,183 +11,214 @@ const chalk = require('chalk')
 const yaml = require('js-yaml')
 const _ = require('lodash')
 
-let fixturesDir = 'fixtures'
-let featureUri
-
 /**
- * Configures the loader
+ * Fixtures loader extension.
  *
- * @param {string} _fixturesDir - The name of the fixtures directory relative to feature
+ * @class
  */
-exports.configure = ({ fixturesDir: _fixturesDir = 'fixtures' } = {}) => {
-    fixturesDir = _fixturesDir
-}
+class FixturesLoader {
+    /**
+     * @param {string} _fixturesDir - The name of the fixtures directory relative to feature
```

Comment:
`@param {string} [_fixturesDir='fixtures'] - The name of the fixtures directory relative to feature`

Reply:
Ok